susemanager-utils/testing/docker/scripts/init-pgsql-reportdb4eclipse.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
```

Comment:
?
```suggestion
set -euxo pipefail
```

Reply:
Not in this PR. Feel free to start an own.

Reply:
ok